manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zheguisoft.IpchatGuiPc
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: Zheguisoft

```

```suggestion
Publisher: Zheguisoft, Inc.
```
This must match Control Panel.